applications/RANSApplication/custom_processes/rans_apply_flag_process.h

```diff
@@ -0,0 +1,130 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_RANS_APPLY_FLAG_PROCESS_H_INCLUDED)
+#define KRATOS_RANS_APPLY_FLAG_PROCESS_H_INCLUDED
+
+// System includes
+#include <string>
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+#include "processes/process.h"
+
+namespace Kratos
+{
+///@addtogroup RANSApplication
+///@{
+
+///@name Kratos Classes
+///@{
+
+/**
+ * @brief Apply a specific flag for nodes and conditions
+ *
+ * This process apply a given flag to nodes of the modelpart.
+ * Then, if preferred, applies to all conditions in the given model, which has
+ * the given flag applied to all the nodes in the specific condition.
+ *
+ */
+
+class KRATOS_API(RANS_APPLICATION) RansApplyFlagProcess : public Process
```

Comment:
```suggestion
class KRATOS_API(RANS_APPLICATION) RansApplyFlagToSkinProcess : public Process
```
To make  it more self-descriptive.

Reply:
done :)